### PR TITLE
Price package key disclaimer

### PR DESCRIPTION
**Pull request by ananthb86**

### CODE REVIEW

File: vipps-partner-api.md [hunk 1]
```diff
@@ -219,6 +219,13 @@ The merchant can not change the information provided by the partner, so if
 something needs to be corrected, the merchant must contact the partner to have
 the partner submit a new product order with the correct details.
 
+Due to how the pricing information for a partner is set up internally, it is important
+for the partner to send in correct information for `pricePakcageKey` in the request.
```
Comment (Haldormyre) [inline vipps-partner-api.md:223]: Typo in `pricePackageKey`

File: docs/swagger.yaml [hunk 2]
```diff
@@ -335,7 +335,7 @@ components:
           minLength: 1
           maxLength: 255
           example: posstandard
-          description: One of the price package that is registered for the partner in Vipps system.
+          description: One of the price package that is registered for the partner in Vipps system. Please note, it is not the PriceId (3 digit) but the name such as standard, price1, etc that is applicable to the partner.
```
Comment (cloveras) [inline docs/swagger.yaml:338]: Add quotes: "standard", "price1"

**Review by cloveras — CHANGES_REQUESTED** [review]
So close!

File: docs/swagger.yaml (hunk 2 above)
Comment (cloveras) [inline docs/swagger.yaml:338]:
```suggestion
          description: One of the price package that is registered for the partner in Vipps system. Please note, it is not the PriceId (3 digit) but the name such as "standard", "price1", etc that is applicable to the partner.
```

File: vipps-partner-api.md (hunk 1 above)
Comment (cloveras) [inline vipps-partner-api.md:223]:
```suggestion
for the partner to send in correct information for `pricePackageKey` in the request. 
```

**Review by cloveras — APPROVED** [review]
LGTM

**Review by cloveras — APPROVED** [review]

**Review by cloveras — APPROVED** [review]

**Pull request merged**